### ISSUE TITLE
false alert about pageroot on admin page

### ISSUE THREAD

**Issue by pridetechdesignllc** [phpList/phplist3#474]
On the admin screen I see an error saying the pageroot is incorrect, and to check the config.

The pageroot is "/", and that is correct (the address is "https://domain.com/")

version is 3.3.8

**Comment by samtuke** [comment]
@oliverklee Thanks -- done

**Comment by samtuke** [comment]
@pridetechdesignllc Thank you for reporting this. Do you know if there is already a [mantis](http://mantis.phplist.org/) issue for it?

**Comment by pridetechdesignllc** [comment]
> @pridetechdesignllc Thank you for reporting this. Do you know if there is already a [mantis](http://mantis.phplist.org/) issue for it?

I don't know, github was the first place I checked and I thought it'd be the most appropriate place to report the issue.

**Comment by bramley** [comment]
@pridetechdesignllc Just noticed this issue, please see this topic in the forum https://discuss.phplist.org/t/solved-pageroot-problem-after-upgrading-to-3-3-7/4817

TL;DR `$pageroot` should be '', an empty string, not '/'.

**Comment by suelaP** [comment]
Resolved: https://github.com/phpList/phplist3/pull/473/

**Comment by pf18387** [comment]
Hi,
The below "warning":
_The pageroot in your config "/lists" does not match the current location "/emma". Check your config file_. 
has been encountered while initializing via
https://localhost/emma/admin/?page=initialise&firstinstall=1&tk=xxx

Contextual item: Apache2 - phpList is defined into a conf-enabled file as:
```
Alias /emma	/srv/datadisk01/webapp/emma/public_html/lists
DocumentRoot	/srv/datadisk01/webapp/emma/public_html/lists
<Directory	/srv/datadisk01/webapp/emma/public_html/lists/>
...
</Directory>
```
Nailed, by setting into <tt>public_html/lists/config/config.php</tt>
<tt>$pageroot			= '/emma<b>/</b>';</tt>

Best regards

-- 
Pierre.
